### PR TITLE
feature/GDEV-151

### PR DESCRIPTION
**Pull request by ac-jorellanaf**

### CODE REVIEW

**Review by MoritzHahn1337 — CHANGES_REQUESTED** [review]
Looks good, once you fixed the checks

**Review by KerstenBreuer — APPROVED** [review]
same here 👍

**Comment by ac-jorellanaf** [comment]
Not sure what the problem is at all.

**Comment by KerstenBreuer** [comment]
The Github actions workflow used the wrong Python version. The chassis lib is only available for 3.9+. Just fixed it.

File: setup.py [hunk 1]
```diff
@@ -32,6 +32,8 @@
     "openapi-core==0.13.8",
     "zope.sqlalchemy==1.5",
     "pydantic==1.8.2",
+    "ghga-service-chassis-lib==0.3.0",
```
Comment (KerstenBreuer) [inline setup.py:35]:
```suggestion
    "ghga-service-chassis-lib[pubsub]==0.3.0",
```

I think we need the pubsub extra don't we?

**Review by KerstenBreuer — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (ac-jorellanaf) [reply setup.py:35]: Yes. It was an attempt at a fix on my end.

File: setup.py [hunk 2]
```diff
@@ -32,6 +32,8 @@
     "openapi-core==0.13.8",
     "zope.sqlalchemy==1.5",
     "pydantic==1.8.2",
+    "ghga-service-chassis-lib[pubsub]==0.3.0",
+    "pika==1.2.0",
```
Comment (KerstenBreuer) [inline setup.py:36]:
```suggestion
```

Oh and this can be removed then, as it's already in the shared lib.

**Review by KerstenBreuer — APPROVED** [review]
Only this minor change

**Comment by KerstenBreuer** [comment]
Can be merged @MoritzHahn1337?

**Review by MoritzHahn1337 — APPROVED** [review]

**Pull request merged**